docker/lr-align/Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION = 0.1.28
+VERSION = 0.1.29
```

Comment:
Can you provide a list of tasks that uses 0.1.29?
It's hard for me to do it; I'd have to go over all your changes to WDL.
That script can help in this case.
Thanks.

Reply:
Nothing actually seems to use `0.1.29`… I'm not sure why I updated the number here.  `0.1.28` is still the latest.  I'll roll this back to `0.1.28`.

Reply:
still to do

Reply:
Done!